ports/stm32/rfcore.c

```diff
@@ -61,23 +81,92 @@ typedef struct _parse_hci_info_t {
     bool was_hci_reset_evt;
 } parse_hci_info_t;
 
-static volatile uint32_t ipcc_mem_dev_info_tab[8];
-static volatile uint32_t ipcc_mem_ble_tab[4];
-static volatile uint32_t ipcc_mem_sys_tab[2];
-static volatile uint32_t ipcc_mem_memmgr_tab[7];
-
-static volatile uint32_t ipcc_mem_sys_cmd_buf[272 / 4];
-static volatile tl_list_node_t ipcc_mem_sys_queue;
-
-static volatile tl_list_node_t ipcc_mem_memmgr_free_buf_queue;
-static volatile uint32_t ipcc_mem_memmgr_ble_spare_evt_buf[272 / 4];
-static volatile uint32_t ipcc_mem_memmgr_sys_spare_evt_buf[272 / 4];
-static volatile uint32_t ipcc_mem_memmgr_evt_pool[6 * 272 / 4];
-
-static volatile uint32_t ipcc_mem_ble_cmd_buf[272 / 4];
-static volatile uint32_t ipcc_mem_ble_cs_buf[272 / 4];
-static volatile tl_list_node_t ipcc_mem_ble_evt_queue;
-static volatile uint32_t ipcc_mem_ble_hci_acl_data_buf[272 / 4];
+// Version
+// [0:3]   = Build - 0: Untracked - 15:Released - x: Tracked version
+// [4:7]   = branch - 0: Mass Market - x: ...
+// [8:15]  = Subversion
+// [16:23] = Version minor
+// [24:31] = Version major
+
+// Memory Size
+// [0:7]   = Flash ( Number of 4k sector)
+// [8:15]  = Reserved ( Shall be set to 0 - may be used as flash extension )
+// [16:23] = SRAM2b ( Number of 1k sector)
+// [24:31] = SRAM2a ( Number of 1k sector)
+
+typedef struct __attribute__((packed)) {
```

Comment:
following convention elsewhere, these struct's should have, eg, `_mb_device_info_table_t` before the `{`